app/src/main/java/com/eveningoutpost/dexdrip/Models/BloodTest.java

```diff
@@ -124,6 +124,10 @@ private BloodTestMessage toMessageNative() {
     private static final long CLOSEST_READING_MS = 30000; // 30 seconds
```

Comment:
Yes, exactly, it is only applied to watch right now - see Home.java line 815.  I didn't change code for the phone.  So, if the user enters a BloodTest on the watch, upon re-connect, app Home will process that data in processAndApproveTreatment() by calling BloodTest.createFromCal, with the new argument, thisuuid.  Watch passes thisuuid to Home in the receivedText and it is parsed in handleWordPair().  It is the same uuid used to create a Treatment, if the user has entered Treatment data as well on the Watch.  Watch also passes a "watchkeypad" to identify the receivedText is coming from the watch, and not phone.  I could have used the new "uuid" field in receivedText but thought it best to have a separate identifier in case in the future uuid is used by phone or other device. 

So the Phone will still  call processCalibrationNoUI() to process new BloodTests which currently displays the UI based on BloodTest prefs, auto, ask, etc.  The watch forgoes this UI and simply calls createFromCal() directly (Home, line 815).  Maybe we still need to display the UI for watch-entered BloodTests?  I thought it simpler to forgo the UI.